### ISSUE TITLE
python version specifier in flit.ini

### ISSUE THREAD

**Issue by d-jeon** [dit/dit#125]
I know the recommended way to install dit is from source code, but at least dev version of dit package is available in central repository now. However, it complains about an invalid specifier.

```
$ pip install dit
Collecting dit
  Using cached dit-1.0.0.dev6-py2.py3-none-any.whl
  Package dit has an invalid Requires-Python entry 2.7,>=3.3 - Invalid specifier: '2.7'
Installing collected packages: dit
Successfully installed dit-1.0.0.dev6
```

I think 2.7 needs comparison operator. Also, I think the comma (",") is equivalent to a logical **and** operator in Requires-Python. So it could be something like:

```
requires-python = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*
```

I hope there are more elegant ways to specify these.

**Comment by Autoplectic** [comment]
thank you! i saw the same problem, but wasn't sure how to fix it.

**Comment by Autoplectic** [comment]
possibly fixed in 87ceee5d35038d3f64e58863dc20b069924c3071
